### DISCUSSION TITLE
A few questions.

### DISCUSSION THREAD

**Discussion by Nanken89** [evilsocket/opensnitch#858] [Development]
I apologize in advance that I distract you.

1: If version 1.6.0 has a built -in blocking of incoming connections, then there is no more need to use gufw firewall?

2: The problem with the delayed reboot (shutdown) on xfce de is still persist?

3: Debian repo have v1.5.5/1.5.7. What versions is this?

4: Version 1.6.X before summer is not worth waiting?

**Comment by gustavo-iniguez-goya** [comment]
Hi @Nanken89 !

> 1: If version 1.6.0 has a built -in blocking of incoming connections, then there is no more need to use gufw firewall?

I haven't reviewed what features gufw offers, but if you're using it only to block inbound connections then maybe opensnitch is enough.

> 2: The problem with the delayed reboot (shutdown) on xfce de is still persist?

I fixed a couple of bugs related with delays, it works much better now, but please, if you see that it still delays the reboot open a new issue.

> 3: Debian repo have v1.5.5/1.5.7. What versions is this?

It's based on branch 1.5.0: https://github.com/evilsocket/opensnitch/tree/1.5.0

Over the last month we've been working to integrate opensnitch in Debian, but there hasn't been significant changes to the functionality. Only translations updates and not much else:
https://github.com/evilsocket/opensnitch/tags

> 4: Version 1.6.X before summer is not worth waiting?

I don't want to add new functionality (only a refactor of something that it's already added), and 1.6.0rc4 seems to be working well , so I hope to release it soon :crossed_fingers: 

I'll release v1.6.0rc5 this week if all goes well, or maybe the next week.
